### PR TITLE
Squishy ball doodle

### PR DESCRIPTION
**Pull request by saniul**
Sunday night doodle: pretty much a straight up port of http://jonobr1.github.io/two.js/examples/rubber-ball.html to prototope

Drag your finger around the screen and the squishy ball will stretch and follow your finger.

### CODE REVIEW

**Comment by saniul** [comment]
This is far from [Squish ’n’ Stretch](https://vimeo.com/93206523) but it still feels pretty fun!

**Comment by andymatuschak** [comment]
Sweet! How did you feel the clarity and quality of what Prototope allowed compares to the two.js implementation?

File: Examples/SquishyBall/main.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	// 1/8 increments of saturation
+    var saturation = Math.max(0.5, Math.random()*(8+1) * 1.0/8.0);
+
+	return new Color({hue: hue, saturation: saturation, brightness: brightness, alpha: 1.0})
+}
+
+Layer.root.backgroundColor = randomPrettyColor();
+
+var delta = new Point();
+var mouse = new Point();
```
Comment (jbrennan) [inline Examples/SquishyBall/main.js]: hehe :)

**Comment by saniul** [comment]
@andymatuschak No new insights, to be honest. At times I really wished things were easier to mutate in-place. Also, performance is pretty poor, which is why the “ball” is only a 15-gon in this example.

File: Examples/SquishyBall/main.js (hunk 1 above)
Comment (saniul) [inline Examples/SquishyBall/main.js]: @jbrennan :)

**Pull request merged**